src/platforms/apple/common/configuration/watchdog-terminations.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Watchdog Terminations
+sidebar_order: 10
+description: "Learn how to turn off Watchdog Termination tracking"
+---
+
+This integration tracks <a href="https://developer.apple.com/documentation/xcode/addressing-watchdog-terminations">watchdog terminations</a> based on heuristics. This feature is available for iOS, tvOS, and Mac Catalyst, works only if the application was in the foreground, and doesn't track watchdog terminations for unit tests.
+
+When a typical unhandled error occurs, the Apple SDK writes a report to disk with the current state of your application, with details like the stack trace, tags, breadcrumbs, and so on, before the app terminates. When the watchdog terminates your app this happens without further notice, which means the SDK can't write a report to disk.
```

Comment:
`m`: I think we should mention that one of the reasons are OOMs, and we don't create a watchdog termination issue if the app was hanging. We create AppHangs for that. Please put that info into the docs here.